midend/interpreter.cpp

```diff
@@ -79,7 +79,7 @@ bool SymbolicValueFactory::isFixedWidth(const IR::Type* type) const {
 }
 
 unsigned SymbolicValueFactory::getWidth(const IR::Type* type) const {
-    type = typeMap->getType(type, true);
+    type = typeMap->getTypeType(type, true);
```

Comment:
This seems unrelated to the rest of the changes. What is this for?

Reply:
Yes, this is correct. This file is in fact not currently used, but these changes were triggered by @VolodymyrPeschanenkoLitSoft
and his changes to the loop unrolling pass. This file is part of loop unrolling, and these changes are indeed fixing some unrelated bugs in this file.